src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -710,24 +710,24 @@ namespace CondenserLoopTowers {
                 tower.DesInletAirDBTemp = 35.0;
                 tower.TowerInletCondsAutoSize = true;
             }
-            tower.DesInletAirWBTemp = NumArray(22);
-            if (tower.DesInletAirWBTemp == 0) {
-                tower.DesInletAirWBTemp = 25.6;
+            tower.DesignInletWB = NumArray(22);
+            if (tower.DesignInletWB == 0) {
+                tower.DesignInletWB = 25.6;
                 tower.TowerInletCondsAutoSize = true;
             }
-            tower.DesApproach = NumArray(23);
-            if (tower.DesApproach == DataSizing::AutoSize || tower.DesApproach == 0) {
-                tower.DesApproach = 3.9;
+            tower.DesignApproach = NumArray(23);
```

Comment:
This is CoolingTower:TwoSpeed, same thing, Approach and Range are incorrectly reported before fix